components/widgets/PrecipitationCurrent/index.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from "prop-types";
+import { useTranslation } from "react-i18next";
+import WidgetBackground from "@/components/atomic/WidgetBackground";
+import * as Styled from "./styles";
+
+const PrecipitationCurrent = ({ precipitation = 0, humidity = 0 }) => {
+  const { t, i18n } = useTranslation();
+  const { language = "en" } = i18n;
+
+  const styledFormatter = (value) => {
```

Comment:
make use of this custom formatter to add styling to the percent sign in keeping with the designs, while also preserving local percentage display